Sasquatch/SasquatchObjC/AppDelegate.m

```diff
@@ -46,6 +46,24 @@ @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
 
+  [MSAppCenter start:@"target=8dfa00f19c154b2ba1308ea49929bc4e-a0715669-ea1e-47d8-928a-dd33841333c5-6753"
```

Comment:
@jwallra This test code broke `setMaxStorageSize` (Unable to set storage size after the application has configured App Center). I guess this was merged by mistake and should be reverted.